roles/chart_verifier/tasks/tests.yml

```diff
@@ -140,7 +136,11 @@
       -v {{ work_chart_dir }}:/app/chartverifier:Z
       {{ chart_verifier_image }}
       verify --set profile.vendorType=partner
-      {% if chart.values_file is defined %} -F /app/chartverifier/{{ chart.values_file | basename }} {% endif %}
+      {%- if chart.values_file is defined %}
```

Comment:
Should we check here that chart.values_file is iterable?

Reply:
Thanks, good idea. Added the validation at the beginning